tasks/django.yml

```diff
@@ -139,6 +139,27 @@
   changed_when: '"\n0 static files" not in collectstatic_output.out'
   tags: collectstatic
 
+- name: set permissions to "{{ virtualenv_path }}"
+  file:
+    path: "{{ virtualenv_path }}"
+    state: directory
+    group: "{{ www_group }}"
+    owner: "{{ www_user }}"
+    # sets executable only to directories
+    mode: u=rwX,g=rX,o=rX
+    recurse: true
+  tags:
+    - molecule-idempotence-notest
```

Comment:
is this really needed?

Reply:
Yes, without this error is shown by ansible: https://github.com/django/django/blob/25cbd1e6aa4178c31d60e3900a5bd4aa177c01c9/django/core/management/commands/compilemessages.py#L149-L155

But on running manually it is not showing errors.

Reply:
Ok, please add this reason as a comment preceding the task